### ISSUE TITLE
First class GraalVM support

### ISSUE THREAD

**Issue by matiwinnetou** [bloxbean/yaci-devkit#42]
Having talked to a new non Java Developers I know that very important for them tend to be LOW memory footprint and "native feeling". We should explore ways how to have **native** only builds on this project. It should be invisible to non java people that this is even written in java in the first place.

**Comment by satran004** [comment]
Agreed. It's been on our to-do list for a long time. However, I think we can start working on it after the 0.8.0 release. Both Yaci CLI and Yaci Store need to be converted to native through GraalVM.

**Comment by satran004** [comment]
Yaci CLI Native Image : https://github.com/bloxbean/yaci-devkit/pull/49
